pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -209,6 +211,58 @@ func (service *SecurityPolicyService) convertNetworkPolicyToInternalSecurityPoli
 			},
 		},
 	}
+	// 1.1 Generating ingress allow rules in allow section
```

Comment:
A nit suggestion, convertNetworkPolicyToInternalSecurityPolicies func is too long, could you break it into smaller functions?

Reply:
done